code/stats/stats.cpp

```diff
@@ -13,253 +13,260 @@
 #include "globalincs/systemvars.h"
 #include "hud/hud.h"
 #include "network/multi.h"
+#include "pilotfile/pilotfile.h"
 #include "playerman/player.h"
 #include "stats/stats.h"
 
-player *Active_player;
+static const scoring_struct *Player_stats;
+static scoring_struct All_time_ever_stats;
 
 void show_stats_init()
 {
 	if (Game_mode & GM_MULTIPLAYER) {				
 		set_player_stats(MY_NET_PLAYER_NUM);
 	} else {
-		Active_player = Player;		
+		Player_stats = &Player->stats;
 	}	
+
+	// multi only has current stats and all-time-ever stats, so this is only needed in single-player
+	Pilot.export_stats(&All_time_ever_stats);
```

Comment:
The `export_stats()` call should be inside the `else` so it only gets called for single player. And `Player_stats` should be pointing to `All_time_ever_stats`.